vignettes/fixest_walkthrough.Rmd

```diff
@@ -471,7 +470,7 @@ In the previous code, `fitstat = ~ . + ivwald2` means that we want to add the se
 Now what about adding some fixed-effects? That's of course possible, you need to add them after the first right-hand-side, as follows:
 
 ```{r}
-est_iv_fe = feols(y ~ x1 | fe | x_endo_1 + x_endo_2 ~ x_inst_1 + x_inst_2, base)
+est_iv_fe = est_iv = feols(qty ~ t | day | price ~ speed2 + wave2, fulton)
```

Comment:
Why the double assignment here?

Reply:
Good catch! Copying and pasting gone wrong